docs/viper-by-example.rst

```diff
@@ -255,16 +255,10 @@ previous examples. Let's dive right in.
 
 .. literalinclude:: ../examples/crowdfund.v.py
   :language: python
-  :lines: 1-7
+  :lines: 1-9
 
 Like other examples, we begin by initiating our variables - except this time,
-we're not calling them with the ``public`` function. Variables initiated this
-way are, by default, private.
-
-..note ::
-Unlike the existence of the function ``public()``, there is no equivalent
-``private()`` function. Variables simply default to private if initiated
-without the ``public()`` function.
+we're not calling them with the ``public`` function. Instead, we call them with the ``private`` function.
```

Comment:
Just this one should be altered tho state that it's implicit private.

Reply:
OK, done.